configure

```diff
@@ -184,6 +192,85 @@ EOF
 cat > $rootdir/config.erl << EOF
 {with_curl, $WITH_CURL}.
 EOF
+ 
+if [ "${WITH_SNAP}" -eq "1" ]; then
+echo "==> configuring snapcraft config"
+cat > rel/snapcraft.yaml << EOF
+name: couchdb
+version: 2.0
```

Comment:
$(COUCHDB_VERSION) was only in Makefile, it wasn't available in ./configure, so I've had to make ./configure load version.mk into envvars